### ISSUE TITLE
Add installation from oci helm 

### ISSUE THREAD

**Issue by hzxuzhonghu** [kmesh-net/website#79]
Since we have released a oci image for helm chart. We can add a guide to help users install kmesh without clone the code repo

**Comment by hzxuzhonghu** [comment]
https://github.com/orgs/kmesh-net/packages/container/package/kmesh-helm

**Comment by Recxsmacx** [comment]
hey @hzxuzhonghu  can you assign this issue to me

**Comment by Recxsmacx** [comment]
Hello! I noticed that we need to add installation instructions for the OCI Helm chart to the documentation. Here’s a draft of the steps I suggest:

**Log in to the OCI registry:**
```bash
 helm registry login ghcr.io

**Pull the helm chart:**
```bash
helm chart pull ghcr.io/kmesh-net/kmesh-helm:v0.5.0

**Export the chart:**
```bash
helm chart export ghcr.io/kmesh-net/kmesh-helm:v0.5.0

**Install the chart:**
```bash
helm install kmesh-release ./kmesh-helm

**please let me know if this looks good or if there are any other details we should add**

**Comment by Recxsmacx** [comment]
Hello! I noticed that we need to add installation instructions for the OCI Helm chart to the documentation. Here’s a draft of the steps I suggest:
### **Log in to the OCI registry:**
` helm registry login ghcr.io`

### **Pull the helm chart:**
`helm chart pull ghcr.io/kmesh-net/kmesh-helm:v0.5.0`

### **Export the chart:**
`helm chart export ghcr.io/kmesh-net/kmesh-helm:v0.5.0`

### **Install the chart:**
`helm install kmesh-release ./kmesh-helm`

**All the commands are in Bash**
**please let me know if this looks good or if there are any other details we should add**

**Comment by hzxuzhonghu** [comment]
I guess `login0` is not a must

and we can also use one cmd `helm install  kmesh --version v0.5.0 oci://ghcr.io/kmesh-net/kmesh-helm -n kmesh-system`

**Comment by Recxsmacx** [comment]
yeah it is making it more clear that login is unnecessary in this case

**Comment by Recxsmacx** [comment]
@hzxuzhonghu hey i have mailed you on your mail. Please do have a look on it.

**Comment by hzxuzhonghu** [comment]
>
@hzxuzhonghu hey i have mailed you on you're mail. Please do have a look on it.


what do you mean? i haven't received anything

**Comment by vaishnavi192** [comment]
Hey! I would like to work on this issue @hzxuzhonghu

**Comment by vaishnavi192** [comment]
Working on it!

**Comment by hzxuzhonghu** [comment]
cool， thank you
